include/eve/conditional.hpp

```diff
@@ -38,6 +38,13 @@ namespace eve
       return or_<C,T>{static_cast<C const&>(*this), v};
     }
 
+    auto map_alternative(auto op) const
```

Comment:
why is this necessary ?

Reply:
Because when we do:

```
load[ignore.else(smth)](converting_iterator) 
```

Or 

```
store[ignore.else(smth)](converting_iterator) 
```

The else part is provided in the type of value we convert to.
And I need to map it to the underlying type.

You will also need this for `load(as<>)` when we have it.

Reply:
OK